From da6f0157de1e88f664351578099a7e538667dd7b Mon Sep 17 00:00:00 2001
From: Ricardo Salveti <ricardo@foundries.io>
Date: Tue, 1 Sep 2020 20:34:40 -0300
Subject: [PATCH 040/322] [FIO temphack] apalis-imx8: set fdt_high to
 0xffffffffffffffff

Set fdt_high to 0xffffffffffffffff in order to avoid copying fdt during
boot, as done by imx8qm_mek.

Such solution can't be upstreamed (see [1]) so remark it as temphack.

[1]
https://patchwork.ozlabs.org/project/uboot/patch/20210722154943.127775-2-oleksandr.suvorov@toradex.com/#2723077

Signed-off-by: Ricardo Salveti <ricardo@foundries.io>
Signed-off-by: Oleksandr Suvorov <oleksandr.suvorov@foundries.io>
---
 include/configs/apalis-imx8.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/configs/apalis-imx8.h b/include/configs/apalis-imx8.h
index 8fe3226cf9d..b09c6499486 100644
--- a/include/configs/apalis-imx8.h
+++ b/include/configs/apalis-imx8.h
@@ -48,6 +48,7 @@
 	MEM_LAYOUT_ENV_SETTINGS \
 	"console=ttyLP1 earlycon\0" \
 	"fdt_addr=0x83000000\0"	\
+	"fdt_high=0xffffffffffffffff\0" \
 	"fdt_file=fsl-imx8qm-apalis-eval.dtb\0" \
 	"fdtfile=fsl-imx8qm-apalis-eval.dtb\0" \
 	"finduuid=part uuid mmc ${mmcdev}:2 uuid\0" \
-- 
2.34.1

